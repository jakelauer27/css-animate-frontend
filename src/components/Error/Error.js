import React, { Component } from 'react'
import { Link } from 'react-router-dom'

class Error extends Component {
  
  render() {
    return (
      <div className='error-page'>
      <h2>page not found</h2>
      <Link to='/'>return to site</Link>
    </div>
    )
  }
}

export default Error