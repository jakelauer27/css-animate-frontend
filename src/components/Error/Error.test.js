import Error from './Error'
import { shallow } from 'enzyme'
import React from 'react'

describe('Error', () => {

  it('should match the snapshot', () => {
    let wrapper = shallow(<Error/>)
    expect(wrapper).toMatchSnapshot()
  })
})