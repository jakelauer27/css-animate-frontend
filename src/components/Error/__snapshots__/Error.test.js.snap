// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Error />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          page not found
        </h2>,
        <Link
          replace={false}
          to="/"
        >
          return to site
        </Link>,
      ],
      "className": "error-page",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "page not found",
        },
        "ref": null,
        "rendered": "page not found",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "return to site",
          "replace": false,
          "to": "/",
        },
        "ref": null,
        "rendered": "return to site",
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            page not found
          </h2>,
          <Link
            replace={false}
            to="/"
          >
            return to site
          </Link>,
        ],
        "className": "error-page",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "page not found",
          },
          "ref": null,
          "rendered": "page not found",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "return to site",
            "replace": false,
            "to": "/",
          },
          "ref": null,
          "rendered": "return to site",
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
